_includes/setup/ios-setup.md

```diff
@@ -48,12 +48,17 @@ iOS devices.
    $ brew install ideviceinstaller ios-deploy
    $ brew install cocoapods
    ```
-   If any of these commands fails with an error, run `brew doctor` and follow the instructions 
+   If any of these commands fails with an error, run `brew doctor` and follow the instructions
    for resolving the issue.
-   
-1. Follow the steps defined by Apple to configure Xcode with your Apple Developer account and enable deploying with provisioning profiles. To learn how to set up your profile, see [Apple's official documentation](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/MaintainingProfiles/MaintainingProfiles.html).
-1. Select your Development Team for the Flutter project.
-   1. Open the default Xcode workspace in your project under `ios/Runner.xcworkspace`. 
-   1. Select the `Runner` project in the left navigation panel.
-   1. In the Target Settings page, make sure your Development Team is selected under **General > Signing > Team**.
+
+1. Follow the Xcode signing flow to provision your project:
+    1. Open the default Xcode workspace in your project by running `open ios/Runner.xcworkspace` in a terminal window from  your Flutter project directory.
```

Comment:
I think we just referred a few more places on the setup page to the project directory assuming the user knows what we're referring to. Do you think we should add it in all the places?